lib/apiservers/engine/backends/volume.go

```diff
@@ -234,7 +242,7 @@ func translateInputsToPortlayerRequestModel(name, driverName string, opts, label
 	model.Metadata = make(map[string]string)
 	model.Metadata[dockerMetadataModelKey] = metadata
 	if err := validateDriverArgs(opts, model); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Bad Driver Arg: %s", err)
```

Comment:
When you look at the docker code for their volume plugin handling they return the raw error codes from the actual VolumeDriver calls. for example: https://github.com/docker/docker/blob/62215920871b8d15a6ffe538baac5abe2b715409/volume/local/local.go#L153

That is the local driver returning filesystem errors. and that gets passed through this section : https://github.com/docker/docker/blob/62215920871b8d15a6ffe538baac5abe2b715409/volume/drivers/proxy.go#L27